docs/nima/sse/sse.adoc

```diff
@@ -0,0 +1,223 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Server-Sent Events in Nima
+:description: Helidon Nima SSE Support
+:keywords: helidon, nima, sse
+:feature-name: SSE
+:rootdir: {docdir}/../..
+
+== Contents
+
+- <<Overview, Overview>>
+- <<Server API, Server API>>
+- <<Client API, Client API>>
+- <<Additional Information, Additional Information>>
+
+== Overview
+
+Server-sent events (SSE) enable servers to push data to clients (e.g. Web browsers) using standard HTTP
+or HTTPS through a unidirectional server-to-client connection. In the server-sent events communication model,
+the client establishes the initial connection, and the server provides the data in the form of
+_event streams_. For more information about server-sent events, see the
+link:https://html.spec.whatwg.org/multipage/server-sent-events.html[`Server-sent events`]
+specification.
+
+Server-sent events is an alternative technology to WebSockets when only server-to-client messaging
+is required and can be accomplished without the need to switch protocols (upgrades) and without
+using imperfect solutions such as long polling. A server-sent connection is typically a long-lived
+connection in which messages are sent to the client over a longer period of time compared to a
+normal request-response connection. It is useful for updating _live data_ such as stock tickers,
+results of live events, etc.
+
+Nima provides support for server and client APIs, although often Web browsers are
```

Comment:
N&iacute;ma provides support for server and client APIs, although Web browsers are popular client alternatives.

`N&iacute;ma`